HandleAjaxCall.php

```diff
@@ -1,5 +1,17 @@
 <?php
-if(!defined('sugarEntry') || !sugarEntry) die('Not A Valid Entry Point');
+
+if (!defined('sugarEntry') || !sugarEntry || !defined('SUGAR_ENTRY') || !SUGAR_ENTRY) {
+    die('Not A Valid Entry Point');
+}
+if (defined('sugarEntry')) {
+    $deprecatedMessage = 'sugarEntry is deprecated use SUGAR_ENTRY instead';
```

Comment:
can this also be translated on all files used?
+    $deprecatedMessage = 'sugarEntry is deprecated use SUGAR_ENTRY instead';